### ISSUE TITLE
Custom syntax highlighter is never called

### ISSUE THREAD

**Issue by illright** [pngwn/MDsveX#99]
## Steps to reproduce:
* Clone the [Svelte template](https://github.com/sveltejs/template)
* Install `mdsvex`: `yarn add -D mdsvex`
* Set up MDsveX in `rollup.config.js`:

```js
// ...
import { mdsvex } from 'mdsvex';

export default {
  // ...
  plugins: [
    // ...
    svelte({
      extensions: ['.svelte', '.svx'],
      preprocess: mdsvex({
        highlight(code, _lang) {
          console.log('called');
          return `<pre>${code}</pre>`;
        }
      }),
    }),
// ...
```
* Add an MDsveX file (`src/test.svx`) with the following content:

````
```js
alert()
```
````

* Import the new file into `src/App.svelte`

## Expected

`called` will be printed anywhere signifying that the highlighter was called

## Actual

Nothing printed, the default highlighter is used

Also I'm wondering if there's any simple way of stripping the initial newline that is rendered in the `<pre>`

**Comment by pngwn** [comment]
Looks like I didn't update the docs. Highlight should be an object that accepts a `highlighter` property:

```js
// ...
import { mdsvex } from 'mdsvex';

function highlighter(code, _lang) {
  console.log('called');
  return `<pre>${code}</pre>`;
}
export default {
  // ...
  plugins: [
    // ...
    svelte({
      extensions: ['.svelte', '.svx'],
      preprocess: mdsvex({
        highlight: { highlighter }
      }),
    }),
// ...
```

I'll update the documentation.

Can you give an example for the initial newline you mentioned?

**Comment by illright** [comment]
The example for the issue would be the default configuration. Follow the same reproduction steps, but don't override the highlighter. Then the code from `test.svx` will be rendered in a `<pre>` element spanning two lines, the first line being just a linefeed and the second containing actual code

**Comment by TheComputerM** [comment]
It looks like this:
![image](https://user-images.githubusercontent.com/24741538/89559165-e9c22280-d832-11ea-8e42-d9ab9f54f91c.png)


\`\`\`js
console.log('hello there')
console.log('general kenobi')
\`\`\`

The default settings were used

**Comment by TheComputerM** [comment]
And the docs are still not updated

**Comment by pngwn** [comment]
Fixed in #110.

Thanks @TheComputerM!
